tests/schema_update_test.go

```diff
@@ -226,6 +227,17 @@ func TestVCSSchemaUpdate(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	license, err := fs.ReadFile(fakeFS, "fake/license")
```

Comment:
Add a comment for this? Not sure why we need to explicitly switch plan here

Reply:
Done. This is copied from method above.